sdk/storage/examples/data_lake_00.rs

```diff
@@ -63,44 +62,16 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     }
 
     println!("getting file system properties...");
-    let get_fs_props_response = file_system.get_properties().execute().await?;
+    let get_fs_props_response = file_system_client.get_properties().execute().await?;
     println!(
         "get file system properties response == {:?}",
         get_fs_props_response
     );
     println!();
 
-    let file_name = "example-file.txt";
```

Comment:
Why is this removed?

Reply:
I moved the file specific operations to the data_lake_01.rs example